recipes/wxpython/bld.bat

```diff
@@ -0,0 +1,14 @@
+:: LOTS more to do here!
```

Comment:
That was left over from trying to actually built it -- but I've given up on that, so I'll clear it out.